### ISSUE TITLE
Add Dictionary Metadata: Reviews

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/Cologne#95]
As I explore a lot of the lexicography context as well, I find quite a few reviews of the dictionaries, like 
```
Reviewed Work: V. S. Apte's The Practical Sanskrit-English Dictionary by P. K. Gode, C. G. Karve
Review by: Ludwik Sternbach
Journal of the American Oriental Society
Vol. 80, No. 1 (Jan. - Mar., 1960), pp. 59-63
Published by: American Oriental Society
DOI: 10.2307/595270
Stable URL: http://www.jstor.org/stable/595270
Page Count: 5
```

I add a draft OCR layer and we could link at least to the JSTOR article from http://www.sanskrit-lexicon.uni-koeln.de/scans/csldoc/index.html  Cologne Sanskrit Lexicon’s documentation, @funderburkjim what do you think? Some points in the reviews answer the questions we try to solve.

**Comment by funderburkjim** [comment]
Sounds like a good idea.

Why don't you take the lead on  how to implement.

**Comment by funderburkjim** [comment]
Currently, the csldoc stuff is awkward to update.  The source is in a folder on my local computer.  
The source is in the 'restructured text' form used by [Sphinx documentation generator](https://en.wikipedia.org/wiki/Sphinx_(documentation_generator)).

When the source is changed, a python program (using sphinx library) generates a static site. Then that
static is uploaded to Cologne from my local computer.

It would be better if this system were in some place where others could improve it, and then some
script at Cologne could install the improved version at Cologne web site.

**Comment by gasyoun** [comment]
@funderburkjim let's add a 5th point under Related material in
http://www.sanskrit-lexicon.uni-koeln.de/scans/SKDScan/2013/web/index.php

http://www.jstor.org/stable/pdf/25207458.pdf Account of the Sabda Kalpa Druma; A Sanskrit Encyclopædical Lexicon by R. Lenz
Journal of the Royal Asiatic Society of Great Britain and Ireland, Vol. 2, No. 1 (1835), pp. 188-200
[Journal]

> It would be better if this system were in some place where others could improve it, and then some
script at Cologne could install the improved version at Cologne web site.

Sure, but I got no clue on how to do it. Maybe @juhnowski or @vvasuki  or @drdhaval2785 is aware of.

**Comment by funderburkjim** [comment]
@gasyoun  The 5th point is added for skd.  Addition is easy, so feel free to submit similar additions for this or other dictionaries as you think appropriate.

**Comment by gasyoun** [comment]
@funderburkjim thanks, more will follow.

**Comment by drdhaval2785** [comment]
@gasyoun Any further reviews you could gather ?

**Comment by gasyoun** [comment]
>  further reviews you could gather

Nope, let's mark as delayed. Just closing all the old issues will not do us a favor.
